commands/usernote.js

```diff
@@ -29,7 +29,7 @@ const warningsToWords = {
 };
 let warningTypes = _.keys(warningsToWords);
 let usageForNerds = 'Usage: .tag [show | add | append] < -n|--note> "<note>" < --user <username> | /u/<username> >\
- [--sub <subreddit> | /r/<subreddit>] [--url <url> | https://<url>] [--color <color>] [--refresh | --refresh-cache]';
+ [--sub <subreddit> | /r/<subreddit>] [--url <url> | https://<url>] [--color <color>] [--refresh | --refresh-cache --all]';
```

Comment:
`--all` should probably be in its own thing?

`[--refresh | --refresh-cache] [--all]`